### PR TITLE
Add comments on Tutorial  for not working on ARM architecture cluster

### PR DESCRIPTION
**Pull request by pegasas**
resolve https://github.com/kubernetes/website/issues/44271

### CODE REVIEW

**Review by adityasamant25 — COMMENTED** [review]
It would be good to have this in a Caution admonition block.
It would be good to have this statement before the "Once you have a [running Kubernetes cluster](https://deploy-preview-44279--kubernetes-io-main-staging.netlify.app/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/)" hyperlink.

**Comment by sftim** [comment]
> It would be good to have this in a Caution admonition block.

Please use `note`, not `caution`. It's frustrating if the tutorial doesn't work but it doesn't put your production apps at risk.

**Comment by pegasas** [comment]
> It would be good to have this statement before the "Once you have a [running Kubernetes cluster](https://deploy-preview-44279--kubernetes-io-main-staging.netlify.app/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/)" hyperlink.

> Please use note, not caution. It's frustrating if the tutorial doesn't work but it doesn't put your production apps at risk.

Thanks for review. 
Done these 2 suggestions.

File: content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html [hunk 1]
```diff
@@ -27,6 +27,9 @@ <h3>Objectives</h3>
 
             <div class="col-md-8">
                 <h3>Kubernetes Deployments</h3>
+
+                <p><strong>Please note that Minikube must be used with the Docker Desktop as the driver for running AMD64 images on ARM architecture.</strong></p>
```
Comment (adityasamant25) [inline content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:31]: nit:
I would suggest to use the Note shortcode.

Consider using
```
{{< note >}}
<p>Minikube must be used with the Docker Desktop as the driver for running AMD64 images on ARM architecture.</p>
{{< /note >}}
```

Comment (pegasas) [reply content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html]: Ack.
I've corrected it into this way.

**Comment by adityasamant25** [comment]
@pegasas The changes look good to me, although I'm not entitled to add the lgtm label.
It would be good to reach out to the reviewers marked in the PR to get a review.

File: content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html [hunk 2]
```diff
@@ -27,6 +27,11 @@ <h3>Objectives</h3>
 
             <div class="col-md-8">
                 <h3>Kubernetes Deployments</h3>
+
+                {{< note >}}
+                <p>Minikube must be used with the Docker Desktop as the driver for running AMD64 images on ARM architecture.</p>
```
Comment (sftim) [inline content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:32]:
```suggestion
                <p>This tutorial uses a container that requires the AMD64 architecture. If you are using
                minikube on a computer with a different CPU architecture, you could try using minikube with
                a driver that can emulate AMD64. For example, the Docker Desktop driver can do this.</p>
```

Comment (pegasas) [reply content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html]: Done.

**Review by sftim — COMMENTED** [review]
Thanks

/lgtm

**Comment by adityasamant25** [comment]
It will be good if we can approve and merge this PR as readers on M-series Macbooks are frequently failing on this tutorial.
Another issue #44756

**Comment by sftim** [comment]
It's a change to the live site. If someone who's not me, and is an org member, adds their /lgtm, then I can approve it.

**Comment by pegasas** [comment]
> It's a change to the live site. If someone who's not me, and is an org member, adds their /lgtm, then I can approve it.
> @pegasas: you cannot LGTM your own PR.

It seems I could not approve my own PR. waiting for owner for merging

**Pull request merged**